### PR TITLE
Add check for R 3.5.0

### PR DESCRIPTION
**Pull request by damianooldoni**
This small PR allows a run of the GitHub Action R CMD CHECK (`./github/R-CMD-check.yaml`) on `ubuntu-latest` with R 3.5.0 installed.
Very curious about the result 🤞  

**NOTE for reviewer: move to https://github.com/inbo/camtraptor/pull/195#issuecomment-1561285732**.

### CODE REVIEW

**Comment by damianooldoni** [comment]
The status is on hold at the moment. If you want to give a try based on your Windows laptop with old R versions, feel free to give a try 👍 
Meanwhile, I converted this PR to draft.

**Comment by damianooldoni** [comment]
I am using RStudio server of ETN to fix this. Via debugging I found easily where things go wrong: `get_scientific_name()`. I hope to solve this right now.

**Comment by damianooldoni** [comment]
`windows-latest` and `macos-latest` fail to load/compile R package `fastmap` with R 3.5.0.
`macos-latest` fails to load/compile R package `terra` with 3.6.3.

However, runs with R 3.6 on `ubuntu-latest` and `windows-latest` demonstrate that the round issue is not OS dependent, but R dependent. With R 3.6.3, regardless of OS, we get:

```r
round(15.1815)
15.181  # instead of 15.182
```

See test failure:
- `windows-latest`: https://github.com/inbo/camtraptor/actions/runs/5069004785/jobs/9102022183#step:6:285
- `ubuntu-latest`:  https://github.com/inbo/camtraptor/actions/runs/5069004785/jobs/9102021444#step:6:285

Replacing 15.1815 with 15.18155 should for sure solve the problem. I will test it.

**Comment by damianooldoni** [comment]
The PR is now ready for review. Thanks to this PR, the package is now running flawlessly on R 3.5. 

The **main issue** has been described above: https://github.com/inbo/camtraptor/pull/195#issuecomment-1543980826.
The final **minor issue** was a rounding issue with R 3.x as described in https://github.com/inbo/camtraptor/pull/195#issuecomment-1561184223.

**Comment by PietrH** [comment]
> `windows-latest` and `macos-latest` fail to load/compile R package `fastmap` with R 3.5.0. `macos-latest` fails to load/compile R package `terra` with 3.6.3.
> 


Do we want to add a note to the documentation, that while R 3.5.0 is supported, we recommend using a more up to date version of R?

File: R/get_species.R [hunk 1]
```diff
@@ -41,7 +41,7 @@ get_species <- function(package = NULL, datapkg = lifecycle::deprecated()) {
       })
     }
     purrr::map_dfr(taxonomy, function(x) {
-      dplyr::tibble(as.data.frame(x))
+      dplyr::tibble(as.data.frame(x, stringsAsFactors = FALSE))
```
[the remark this answers is not in the document]
Comment (PietrH) [reply R/get_species.R:44]: Seems to work: https://github.com/inbo/camtraptor/actions/runs/5080196156/jobs/9126857519?pr=210

Which do you find easiest to read?

**Review by PietrH — APPROVED** [review]
I'd suggest a small note in the [installation documentation](https://inbo.github.io/camtraptor/#installation) that while we support older versions of R, we recommend using R 4.0.0 or higher (although we could just recommend using the newest version just to play it safe). 

You can merge this without the documentation change, but perhaps we'll make someone's life easier in the future just by mentioning it now. Maybe even our future selves! 

I've also suggested an alternative solution for `dplyr::tibble(as.data.frame( `, but please choose whichever solution you find easiest to read.

Well done for helping camtraptor run on R3.5.0, I appreciate it!

File: R/get_species.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (damianooldoni) [reply R/get_species.R:44]: `purrr` forever 😄

**Pull request merged**